h/static/scripts/directive/form-validate.coffee

```diff
@@ -2,15 +2,15 @@ module.exports = ->
   controller: ->
     controls = {}
 
-    addControl: (control) ->
+    this.addControl = (control) ->
```

Comment:
Not sure I understand this change. From here it looks like controller is just a function, not a constructor, so does this even work?